gpgs/api/gpgs.script_api

```diff
@@ -281,6 +281,8 @@
                     - `gpgs.ERROR_STATUS_SNAPSHOT_FOLDER_UNAVAILABLE`
 
                     - `gpgs.ERROR_STATUS_SNAPSHOT_NOT_FOUND`
+					
```

Comment:
The indentation here looks weird. Could you fix this please and then it's ready to merge!